src-electron/generator/helper-endpointconfig.js

```diff
@@ -93,6 +93,20 @@ function endpoint_fixed_profile_id_array(options) {
   return '{ ' + profileIds.join(', ') + ' }'
 }
 
+/**
+ * Creates array of parent endpoint identifier fields on endpoints
+ *
+ *
+ * @returns C array including the { } brackets
+ */
+function endpoint_fixed_parent_id_array() {
```

Comment:
Add a block helper which does not depend on language. Leave the language to be decided in the template.

Reply:
okay thanks